cowprotocol/cowswap
# [EthFlow]#1297 refactor cancellation modal

# Summary

Part of #1297 

Refactoring current soft cancellation modal

**Note:** Change not directly related to ETH flow, so no need to enable it to test it

# To Test
1. Place regular order
2. Cancel it
* Should work as usual
3. Place limit order
4. Cancel it
* Should work as usual
5. Place an order that's out of market
6. When the 'cancel' button appear, cancel it
* Should work as usual
